### DISCUSSION TITLE
can we retrieve source and destination vertex properties while using "FETCH" for retrieving edge properties?

### DISCUSSION THREAD

**Discussion by susanth19** [vesoft-inc/nebula#4483] [Q&A]
Hi All,

FETCH PROP ON SERVE  "123"->"456" yield properties(edge) .
on the above query can we retrieve the properties of the source vertex("123") and destination vertex("456")  using "FETCH"? is there any way to get these vertex properties using "FETCH"?

Thanks

**Comment by xiajingchun** [comment]
Unlikely, FETCH is not really designed for that. 
Could you use GO or MATCH instead? Or would you mind ot share with us your actual scenario? Thanks.
